index.html

```diff
@@ -37,37 +37,157 @@ <h3>A small web firm in Norwich</h3>
 					<section>
 					<h1>Customers</h1>
 					<aside class="notes">
-					<h3>Started with Channel 4, since then a bunch of charities, slew of cottage rental agencies, Comic Relief, a big American airline, and the World Wide Web Foundation (It's where Tim Berners Lee works.)</h3>
+					<h3>Started with Channel 4, since then a load of tech-for-good work, a bunch of projects funded by Comic Relief, a big American airline, and a bit of design work for the World Wide Web Foundation (It's where Tim Berners Lee works.)</h3>
 					</aside>
 					</section>
 
 				</section>
-
+				
 				<section>
 				<h1>A "lightning talk"</h1>
 				<aside class="notes">
-				Called "Objectives, characters and scenarios" - meaningful ideas out of the aether. A twenty minute lightning talk explaining how our bit of the industry starts to think about what a digital thing should do.
+				<h3>Called "Personas in Practise". A twenty minute lightning talk explaining how our bit of the industry uses personas to help us design and develop useful software.</h3>
```

Comment:
I don't think you can have a 20 minute "lightning talk"

And anyway... Isn't this a 3 hour workshop?